doc/development.rst

```diff
@@ -326,6 +332,16 @@ A root state is a state that has no dependencies.
         graph_strategy.transition('B')  #  returns: []
         graph_strategy.transition('B', via=['A2'])  #  returns: ['Root', 'A2', 'B']
 
+Use `pytest fixtures <https://docs.pytest.org/en/latest/fixture.html>`_ to
+`transition()` into the designated state before a test is executed.
+Use `transition('my_state', via=...)` to follow a (non-default) path, e.g. a
+shortcut.
+This can be achieved by setting an instance variable in a state of a Graph
+Strategy (e.g. during board provisioning) and letting the fixture decide which
+path to follow depending on that variable.
+This variable must be set to its initial value in the `invalidate()` method of
+a Graph Strategy.
```

Comment:
Hmm, I'm still against moving logic (deciding which path to take) from the strategy to a fixture. The fixture should only request a state and the strategy should find the (best) path to that state. The idea described in the next commit seems to follow that separation more cleanly. So we should just describe how to use the current graph strategy here and not how to work around it.

Reply:
The actual information which path to follow is currently always in the fixture, because the fixture calls `transition(.., via=..)`. There is no other logic in the GraphStrategy to follow a better path if not explicitly defined with `via=`. Or am I misunderstanding you?